pkg/apis/logging/v1alpha1/types.go

```diff
@@ -24,46 +24,66 @@ type ClusterLogging struct {
 
 type ClusterLoggingSpec struct {
   AllInOne          bool `json:allinone`
```

Comment:
We are removing this right?

Reply:
Did we come to a conclusion on how we would accomplish telling the cluster to send logs to two different destinations (and stand up the required LogStores if necessary)? To me, this says we need to have multiple Visualizations, LogStores, and Curations and the Collection is going to need to use different end points (as opposed to them being the same)

Reply:
If we are unable to do a separate operations deployment for 4.0 though, would that be considered a regression? 

The reason I don't think being able to have named pipelines would be enough is that we don't have a way to say "give me two different ES clusters" and that was what `AllInOne` was meant to accomplish.

Reply:
Per the discussion on slack, going to make this functionality be provided by an annotation instead